Formula/testkube.rb

```diff
@@ -26,6 +26,9 @@ def install
 
     system "go", "build", *std_go_args(output: bin/"kubectl-testkube", ldflags: ldflags),
       "cmd/kubectl-testkube/main.go"
+
+    bin.install_symlink "kubectl-testkube" => "tk"
```

Comment:
This will make it conflict with `tanka`:

https://github.com/Homebrew/homebrew-core/blob/b2b9082b41e9324bd173d78b08dacd7dc36eaed2/Formula/tanka.rb#L34

Reply:
Removed `tk`